docs/document/content/user-manual/shardingsphere-proxy/cdc/build.cn.md

```diff
@@ -0,0 +1,224 @@
++++
+title = "运行部署"
+weight = 1
++++
+
+## 背景信息
+
+ShardingSphere CDC 分为两个部分，一个是 CDC Server，另一个是 CDC Client。 CDC Server 和 ShardingSphere-Proxy 目前是一同部署的。
+
+用户可以在自己的项目中引入 CDC Client，实现数据的消费逻辑。
+
+## 约束条件
+
+- 纯 JAVA 开发，JDK 建议 1.8 或以上版本。
+- CDC Server 要求 SharingSphere-Proxy 使用集群模式，目前支持 ZooKeeper 作为注册中心。
+- CDC 只同步数据，不会同步表结构，目前也不支持 DDL 的语句同步。
+- CDC 增量阶段会按照事务的维度输出数据， 如果要开启 XA 事务的兼容，则 openGauss 和 ShardingSphere-Proxy 都需要 GLT 模块
+
+## CDC Server 部署步骤
+
+这里以 openGauss 数据库为例，介绍 CDC Server 的部署步骤。
+
+由于 CDC Server 内置于 ShardingSphere-Proxy，所以需要获取 ShardingSphere-Proxy。详情请参见 [proxy 启动手册](/cn/user-manual/shardingsphere-proxy/startup/bin/)。
+
+### 配置 GLT 模块（可选）
+
+官网发布的二进制包默认不包含 GLT 模块，不保证跨库事务完整性，如果使用的是包含 GLT 功能的 openGauss 数据库，则可以额外引入 GLT 模块，保证跨库事务的完整性。
+
+目前有两种方式引入 GLT 模块，并且需要在 server.yaml 中也进行相应的配置。
+
+#### 1. 源码编译安装
+
+1.1 准备代码环境，提前下载或者使用 Git clone，从 Github 下载 [ShardingSphere](https://github.com/apache/shardingsphere.git) 源码。
+
+1.2 删除 kernel/global-clock/type/tso/core/pom.xml 中 shardingsphere-global-clock-tso-provider-redis 依赖的 `<scope>provided</scope>` 标签和 kernel/global-clock/type/tso/provider/redis/pom.xml 中 jedis
+   的 `<scope>provided</scope>` 标签
+
+1.3 编译 ShardingSphere-Proxy，具体编译步骤请参考 [ShardingSphere 编译手册](https://github.com/apache/shardingsphere/wiki#build-apache-shardingsphere)。
+
+#### 2. 直接引入 GLT 依赖
+
+可以从 maven 仓库中引入
+
+2.1. [shardingsphere-global-clock-tso-provider-redis](https://repo1.maven.org/maven2/org/apache/shardingsphere/shardingsphere-global-clock-tso-provider-redis)，下载和 ShardingSphere-Proxy 同名版本
+
+2.2. [jedis-4.3.1](https://repo1.maven.org/maven2/redis/clients/jedis/4.3.1/jedis-4.3.1.jar)
+
+### CDC Server 使用手册
+
+1. 修改配置文件 `conf/server.yaml`，打开 CDC 功能。 目前 `mode` 必须是 `Cluster`，需要提前启动对应的注册中心。如果 GLT provider 使用 Redis，需要提前启动 Redis。
+
+配置示例：
+
+1. 在 `server.yaml` 中开启 CDC 功能。
+
+```yaml
+mode:
+  type: Cluster
+  repository:
+    type: ZooKeeper
+    props:
+      namespace: cdc_demo
+      server-lists: localhost:2181
+      retryIntervalMilliseconds: 500
+      timeToLiveSeconds: 60
+      maxRetries: 3
+      operationTimeoutMilliseconds: 500
+
+authority:
+  users:
+    - user: root@%
+      password: root
+  privilege:
+    type: ALL_PERMITTED
+
+# 使用 GLT 的时候也需要开启分布式事务
+#transaction:
+#  defaultType: XA
+#  providerType: Atomikos
+#
+#globalClock:
+#  enabled: true
+#  type: TSO
+#  provider: redis
+#  props:
+#    host: 127.0.0.1
+#    port: 6379
+
+props:
+  system-log-level: INFO
+  check-table-metadata-enabled: false
+  proxy-default-port: 3307 # Proxy default port.
+  cdc-server-port: 33071 # CDC Server 端口，必须配置
+  #proxy-frontend-database-protocol-type: openGauss # 和后端数据库的类型一致
```

Comment:
Could we uncomment `globalClock` and `proxy-frontend-database-protocol-type`, if they're required